### PR TITLE
code: improve rewriter test coverage

### PR DESCRIPTION
**Pull request by Deardrops**
### What problem does this PR solve? 

Improve test coverage.

### What is changed and how it works?

Improve test coverage.

### Check List 

Tests 

 - Unit test

Side effects

N/A

Related changes

N/A

### CODE REVIEW

**Comment by Deardrops** [comment]
I'm still working on improving  test coverage.
please help me trigger the `Codecov Report` in this PR, thanks

**Comment by lonng** [comment]
@xiekeyi98 PTAL

**Comment by xiekeyi98** [comment]
It seems we could see the coverage report in  https://codecov.io/github/pingcap/failpoint/commit/bea37269311c3f57ec867dc11f32f8d937293f39
I'm not sure why the codecov bot does not comment in this PR.

**Comment by xiekeyi98** [comment]
I think may the packets loss leads to the bot comment loss, we may rerun the Travis-CI.

File: code/rewriter.go [hunk 1]
```diff
@@ -159,9 +160,7 @@ func (r *Rewriter) rewriteExpr(expr ast.Expr) error {
 		return r.rewriteFuncLit(ex)
 
 	case *ast.CompositeLit:
-		if err := r.rewriteExpr(ex.Type); err != nil {
```
Comment (Deardrops) [inline code/rewriter.go]: This case is needless, I remove them because the `Type` cannot be the `Expression`, We don't need to check and rewrite it.

**Comment by Deardrops** [comment]
Finished, @lonng PTAL

File: code/rewriter.go (hunk 1 above)
Comment (lonng) [reply code/rewriter.go]: OK

**Comment by lonng** [comment]
LGTM
Ping @kennytm

**Pull request merged**